reliability-engineering/terraform/modules/concourse-secrets-admin/variables.tf

```diff
@@ -0,0 +1,25 @@
+data "aws_caller_identity" "account" {}
+
+variable "deployment" {
+  type = string
+}
+
+variable "name" {
+  type = string
+}
+
+variable "kms_key_arn" {
+  type = string
+}
+
+variable "connector_host_path" {
+  type = string
+}
+
+variable "client_id" {
```

Comment:
Suggest calling this `github_oauth_client_id`

Reply:
accepted and embodied.